ortools/sat/csharp/IntegerExpressions.cs

```diff
@@ -69,6 +69,21 @@ public static void AddOrIncrement(this Dictionary<IntVar, long> dict, IntVar key
             list.Capacity = Math.Max(list.Count + collection.Count, list.Capacity);
         }
     }
+
+#if NETFRAMEWORK
```

Comment:
which version NETFRAMEWORK covers ? We will soon support only net6 and above.

Reply:
All of them technically, but since as far as I saw it's currently only built for .NET 4.5.2, Core 3.1 and .NET 6. So it is only used for the first one, the last 2 do have this API on the Queue class, so they don't need it. [See here](https://docs.microsoft.com/en-us/dotnet/api/system.collections.generic.queue-1.trydequeue?view=net-6.0)